package.json

```diff
@@ -55,18 +55,25 @@
         "stylelint-formatter-pretty": "^2.1.1",
         "stylelint-no-unsupported-browser-features": "^4.0.0",
         "stylelint-selector-bem-pattern": "^2.1.0",
-        "stylelint-webpack-plugin": "^2.1.1"
+        "stylelint-webpack-plugin": "^2.1.1",
+        "nx": "14.1.9",
+        "@nrwl/nx-cloud": "latest"
     },
     "scripts": {
+        "build:all": "nx run-many --target=build",
+        "build:one": "f () { nx build @deriv/$1 $2 ;}; f",
+        "build:since": "nx affected --target=build",
+        "test:eslint-all": "nx run-many --target=test:eslint",
+        "test:eslint-one": "f () { nx test @deriv/$1 --target=test:eslint ;}; f",
+        "test:eslint-since": "f () { nx affected --target=eslint ;}; f",
         "bootstrap": "f () { npm ci && lerna link && lerna bootstrap --ci --hoist --strict && lerna link ;}; f",
         "bootstrap:dev": "f () { npm ci && lerna link && lerna bootstrap --hoist --strict && lerna link ;}; f",
-        "build": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run build $2 ;}; f",
         "build:publish": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run build:publish $2 ;}; f",
-        "build:local": "f () { lerna exec --scope @deriv/trader --scope @deriv/bot-web-ui --parallel -- npm run build $1 ;}; f",
+        "build:local": "f () { nx run-many --target=build --projects=@deriv/bot-web-ui,@deriv/trader,$1 --parallel=2 ;}; f",
         "build:travis": "lerna exec --scope @deriv/shared --scope @deriv/components --scope @deriv/translations  --scope @deriv/cashier --scope @deriv/account --scope @deriv/p2p --scope @deriv/cfd --scope @deriv/reports -- npm run build:travis",
-        "build:prod": "export NODE_ENV=staging && npm run build && export NODE_ENV=",
+        "build:prod": "export NODE_ENV=staging && npm run build:all && export NODE_ENV=",
         "build:storybook": "cd packages/components && build-storybook --output-dir .out",
-        "build:gh-pages": "f () { lerna exec --scope @deriv/components --scope @deriv/p2p -- npm run build && lerna exec --scope @deriv/cashier --scope @deriv/account --scope @deriv/cfd --scope @deriv/reports -- npm run build $1 && npm run build:local $1 ;}; f",
+        "build:gh-pages": "f () { nx run-many --target=build projects=@deriv/components,@deriv/p2p && nx run-many --target=build --projects=@deriv/cashier,@deriv/account,@deriv/cfd,@deriv/reports && npm run build:local $1 ;}; f",
```

Comment:
Is there a reason for the 2 statements separated by &&, why can't we combine them?

Reply:
as far as i understood i think they wanted to first build components and p2p and then the rest. 
not sure about it and just kept current configuration and changed the lerna scripts with nx